.travis.yml

```diff
@@ -1,18 +1,24 @@
 language: python
+sudo: required
+dist: trusty
 before_install:
   - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
   - sudo apt-get -qq update
-  - sudo apt-get install gcc-4.8 -y --force-yes
+  - sudo apt-get install gcc-4.8 g++-4.8 -y --force-yes
+  - sudo apt-get install -y libsnappy-dev zlib1g-dev libbz2-dev libgflags-dev
   - ls /usr/lib/gcc/x86_64-linux-gnu/4.8/include/
 install:
   - pip install --upgrade pip
+  - ./scripts/install_pyrocksdb.sh $HOME/src-dependencies 4.8 0.4
   - pip install -r requirements.txt
-  - pip install pylint>1.6.1
+  - pip install 'pylint>=1.6.1'
```

Comment:
I'm not sure... I was flailing around trying to get things working on travis when I did that.  although when I ran that command locally, I ended up with a file named `1.6.1` in my working dir 😄